modules/fixtures/postgres-client.yml

```diff
@@ -9,6 +10,7 @@ spec:
     backoffLimit: 0
     template:
         spec:
+            serviceAccountName: aurora-access-sa
             restartPolicy: Never
             containers:
                 - name: postgres-client
```

Comment:
can't select the line below, but essentially could consider using the same `amazonlinux:latest` for this as well to have those "irsa" related stuff aligned.